both/access_control.js

```diff
@@ -231,101 +229,75 @@ if(Meteor.isServer) {
         'createdAt',
       ];
 
-      if(_.difference(fields, allowedFields).length > 0) {
+      return _onlyAllowedFields(fields, allowedFields);
```

Comment:
Yeah, that is a good and useful pattern. But it's not the *only* pattern.

I've heard the debugger argument before, but I wouldn't trade readability for debugging ease.
 Not that I've ever used a JS debugger.

What do you think about the "private functions starts with underscore" convention? And is that the right place and manner to declare it in?